src/app/plugins/githubuploadadapter.js

```diff
@@ -129,10 +129,14 @@ export class Adapter {
 						this._sendRequest( data );
 					} ) )
 					.then( ( /* { file, response } */ ) => {
```

Comment:
Since handling here became very messy I took the liberty to extract all of these 3 promise handlers to 3 separate functions with decent names so that it's more obvious for people what they do, and so that they became somewhat more separated.